src/ui/hexedit.cc

```diff
@@ -114,8 +119,10 @@ HexEdit::HexEdit(FileBlobModel *dataModel, QItemSelectionModel *selectionModel,
       current_area_(WindowArea::HEX),
       cursor_pos_in_byte_(0),
       cursor_visible_(false),
+      selectionColor_(viewport()->palette().color(QPalette::Highlight)),
```

Comment:
The issue is that order of initialization is different than order of fields in class.